modules/appendix/examples/social_net/select_statement_index_where_multiple_edge_type.gsql

```diff
@@ -1,7 +1,7 @@
 CREATE QUERY multiple_edge_type_where_ex(VERTEX<Person> m1) FOR GRAPH Social_Net {
     all_user = {m1};
     filtered_user = SELECT s
-        FROM all_user:s - ((Posted|Liked|Friend):e) - (Post|Person):t
+        FROM (s:all_user) -[e:Posted|Liked|Friend]-> (t:Post|Person)
```

Comment:
Posted and Liked are directed edge types -[ ]->
Friend is an undirected edge type -[ ]-
Please check the documentation for what is the right way to handle a mixed set of edge types.